content/en/docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -78,9 +78,8 @@ unless the Pod's grace period expires. For more details, see
 [Pod Lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/).
 
 {{< note >}}
-Kubernetes only sends the preStop event when a Pod is *terminated*.
+Kubernetes only sends the preStop event when a Pod or a container in the Pod is *terminated*.
```

Comment:
this change is 👍

Reply:
`executePreStopHook()` is called in `killContainer()` as

https://github.com/kubernetes/kubernetes/blob/d2504c94a0116d4432a8a73fc17a0ec8d003d37b/pkg/kubelet/kuberuntime/kuberuntime_container.go#L697

so this change seems good for me.